internal/gtfs/direction_calculator.go

```diff
@@ -77,7 +77,7 @@ func (dc *DirectionCalculator) calculateFromShape(ctx context.Context, stopID st
 			directions[direction]++
 		}
 	}
-
+	// Return the most common direction found in the directions map
```

Comment:
This comment doesn't add any value. It feels like it was generated by an overeager generative AI system.